ui/app/controllers/allocations/allocation/task/logs.js

```diff
@@ -0,0 +1,24 @@
+import { alias } from '@ember/object/computed';
+import Controller, { inject as controller } from '@ember/controller';
+import { computed } from '@ember/object';
+export default Controller.extend({
+  taskController: controller('allocations.allocation.task'),
+  breadcrumbs: alias('taskController.breadcrumbs'),
+  network: alias('model.resources.networks.firstObject'),
+  ports: computed('network.reservedPorts.[]', 'network.dynamicPorts.[]', function() {
+    return (this.get('network.reservedPorts') || [])
+      .map(port => ({
+        name: port.Label,
+        port: port.Value,
+        isDynamic: false,
+      }))
+      .concat(
+        (this.get('network.dynamicPorts') || []).map(port => ({
+          name: port.Label,
+          port: port.Value,
+          isDynamic: true,
+        }))
+      )
+      .sortBy('name');
+  }),
```

Comment:
Looks like this was copied over from the `task/index` controller. Is `ports` used anywhere on the logs page?